### PR TITLE
just added few times and changes to be made

### PR DESCRIPTION
**Pull request by Manan-dev**
Don't approve this until we have everything finalized, just put it here so i can work on other stuff

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -298,38 +298,38 @@ <h3 class="table-header">Saturday, October 30th</h3>
...
-                <td class="middle">TBA</td>
+                <td class="middle">6th floor Commons</td>
                 <td class="right">12 AM</td>
             </tr>
 
             <tr>
                 <td class="left">Breakfast</td>
-                <td class="middle">TBA</td>
+                <td class="middle">6th floor Commons</td>
                 <td class="right">TBA</td>
```
Comment (andreicozma1) [inline index.html:308]: Could we add the finalized times for meals from Franklin?

File: index.html [hunk 2]
```diff
@@ -298,38 +298,38 @@ <h3 class="table-header">Saturday, October 30th</h3>
         <table class="table">
             <tr>
                 <td class="left">Midnight Snack</td>
-                <td class="middle">TBA</td>
+                <td class="middle">6th floor Commons</td>
                 <td class="right">12 AM</td>
```
Comment (andreicozma1) [inline index.html:302]: Convert all times to the same format for consistency, like 12:00AM if minutes are missing from any

File: index.html [hunk 3]
```diff
@@ -298,38 +298,38 @@ <h3 class="table-header">Saturday, October 30th</h3>
...
                 <td class="middle">TBA</td>
-                <td class="right">TBA</td>
+                <td class="right">2:00 PM</td>
             </tr>
 
             <tr>
                 <td class="left">Sponsor Tech Talk</td>
                 <td class="middle">TBA</td>
-                <td class="right">TBA</td>
+                <td class="right">3:00 PM</td>
```
Comment (andreicozma1) [inline index.html:332]: What should we do about the other sponsor tech talks times? For Saturday should we add the other available times as well even tho not all may be filled out?

**Review by andreicozma1 — CHANGES_REQUESTED** [review]

**Comment by andreicozma1** [comment]
Let's finish these up at the meeting as priority and have the second one as lesser priority @Manan-dev 
CC: @hunterprice04

**Comment by Manan-dev** [comment]
Let me know what else I should add to schedule.

File: index.html [hunk 4]
```diff
@@ -247,48 +247,48 @@ <h3 class="table-header">Friday, October 29th</h3>
...
-                <td class="middle">TBA</td>
-                <td class="right">TBA</td>
+                <td class="middle">Room 435</td>
+                <td class="right">8:30 PM</td>
             </tr>
             <tr>
                 <td class="left">Sponsor Tech Talk</td>
-                <td class="middle">TBA</td>
-                <td class="right">TBA</td>
+                <td class="middle">Room 435</td>
```
Comment (andreicozma1) [inline index.html:284]: Remove room numbers from tech talks for now as we still gotta confirm those 100%, as of now they are tentative but for the public let's keep TBA

**Review by andreicozma1 — APPROVED** [review]
Just the change mentioned above, otherwise everything looks great!

**Review by andreicozma1 — CHANGES_REQUESTED** [review]
Just change mentioned above, otherwise everything looks great!

**Review by andreicozma1 — APPROVED** [review]

**Pull request merged**